src/charts/tooltip.js

```diff
@@ -677,22 +701,22 @@ define(function(require){
         };
 
         /**
-         * Gets or Sets shouldShowDateInTitle
+         * Gets or Sets shouldAppendDateToTitle
          * @param  {Boolean} _x           Desired value
-         * @return {Boolean | module}    Current shouldShowDateInTitle or Chart module to chain calls
+         * @return {Boolean | module}    Current shouldAppendDateToTitle or Chart module to chain calls
          * @public
          */
-        exports.shouldShowDateInTitle = function(_x) {
+        exports.shouldAppendDateToTitle = function(_x) {
```

Comment:
It seems to be duplicated!

Reply:
NO IDEA how/why that happened, but removed it in 206e08e37f9cd7f47a7c00540a34d1d62e551bd3.